eg-next/examples/primitives-fill.rs

```diff
@@ -24,7 +24,7 @@ where
         .into_styled(style)
         .draw(target)?;
 
-    Rectangle::new(Point::new(96, 0), Size::new(64, 64))
+    Rectangle::new(Point::new(97, 0), Size::new(64, 64))
```

Comment:
Why is this change necessary? (Maybe related to the comment below)

Reply:
Well spotted, yes that was to fix some clipping. I've now fixed the new clipping this changed caused so we should be all good now.